Library/Formula/geoipupdate.rb

```diff
@@ -25,7 +25,7 @@ def install
     # See https://github.com/maxmind/geoip-api-c#150
     inreplace "conf/GeoIP.conf.default", "YOUR_USER_ID_HERE", "999999"
     inreplace "conf/GeoIP.conf.default", "YOUR_LICENSE_KEY_HERE", "000000000000"
-    inreplace "conf/GeoIP.conf.default", /^ProductIds .*$/, "ProductIds 506 533"
+    inreplace "conf/GeoIP.conf.default", /^ProductIds .*$/, "ProductIds 506 533 GeoLite2-City GeoLite2-Country"
```

Comment:
This adds the [GeoLite2 databases](http://dev.maxmind.com/geoip/geoip2/geolite2) used by libmaxminddb (and all the newer MaxMind APIs) to the default configuration. 506 and 533 are GeoLite Legacy.